### PR TITLE
[BarClerk-947] - [Markup] Create Client Profile page

### PR DESCRIPTION
**Pull request by jasonclchua**
## Issue Link
- https://app.asana.com/0/1203234368773394/1203256371568947

## Definition of Done
- [x] Client Info component
- [x] GOL component
- [x] Runsheet component
- [x] Court Appearances component

## Pre-condition
- 

## Expected Output
- When a user clicks on an existing Matter in the Dashboard, they will be redirected to the Client Profile page with the client's information.

## Screenshots/Recordings
image

### CODE REVIEW

**Comment by jsvelte** [comment]
Please install `prettier` in your vscode with auto save format

**Review by impaulintech — APPROVED** [review]

File: client/src/components/molecules/ClientProfileCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import { useRouter } from 'next/router'
+import React, { FC, useState } from 'react'
+import SeeMore from '~/components/atoms/SeeMore';
+import { IClientProfile } from '../../../shared/interfaces/index';
```
Comment (impaulintech) [inline client/src/components/molecules/ClientProfileCard/index.tsx]: import { IClientProfile } from '~/shared/interfaces/index';

File: client/src/shared/css/globals.css [hunk 2]
```diff
@@ -8,3 +8,9 @@
   padding-bottom: 7px;
   margin-top: -7px;
 }
+.multiline-truncate {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3;
+  overflow: hidden;
+}
```
Comment (impaulintech) [inline client/src/shared/css/globals.css]: add new line

File: client/src/components/molecules/ClientProfileCard/index.tsx (hunk 1 above)
Comment (jasonclchua) [reply client/src/components/molecules/ClientProfileCard/index.tsx]: done

File: client/src/shared/css/globals.css (hunk 2 above)
Comment (jasonclchua) [reply client/src/shared/css/globals.css]: Unused. Removed changes.

**Pull request merged**